### ISSUE TITLE
_init.js failling. PolyIO.gClient is not a function.

### ISSUE THREAD

**Issue by manuchis** [kengz/aiva#18]
Hello, I'm trying to make it work but I have this error that I couldn't find out where comes from: 

`[Wed Oct 19 2016 13:44:58 GMT+0200 (CEST)] ERROR Unable to load /home/user/GitHub/aiva/scripts/_init: TypeError: polyIO.Client is not a function`

I think that configuration is working well, debug before that line is: 

```
[Wed Oct 19 2016 13:44:53 GMT+0200 (CEST)] INFO Starting poly-socketio server on port: 6466, expecting 3 IO clients
[Wed Oct 19 2016 13:44:53 GMT+0200 (CEST)] DEBUG adapterConfig {
  "ACTIVATE": true,
  "HUBOT_SLACK_TOKEN": "-mytoken-"
}
[Wed Oct 19 2016 13:44:53 GMT+0200 (CEST)] DEBUG import js lib from client.js
[Wed Oct 19 2016 13:44:53 GMT+0200 (CEST)] INFO Starting socketIO client for js at 6466
[Wed Oct 19 2016 13:44:53 GMT+0200 (CEST)] INFO Starting socketIO client for python3 at 6466
[Wed Oct 19 2016 13:44:53 GMT+0200 (CEST)] DEBUG Set SLACK PORT: 8345
[Wed Oct 19 2016 13:44:53 GMT+0200 (CEST)] DEBUG No WEBHOOK set for adapter SLACK
[Wed Oct 19 2016 13:44:53 GMT+0200 (CEST)] INFO Deploying bot aivadev with adapter SLACK
[Wed Oct 19 2016 13:44:53 GMT+0200 (CEST)] DEBUG js YjBROj4Z-tFL-zDOAAAA joined, 2 remains
[Wed Oct 19 2016 13:44:54 GMT+0200 (CEST)] DEBUG py 2q3FQ61SQCHDERP8AAAB joined, 1 remains
[Wed Oct 19 2016 13:44:56 GMT+0200 (CEST)] INFO Logged in as bot1_test_cs of Geo-C
[Wed Oct 19 2016 13:44:57 GMT+0200 (CEST)] INFO Slack client now connected
```

I hope you can help me. thanks

**Comment by kengz** [comment]
Oh hey, a breaking update the `polyIO` dependency broke it. Just updated the master and retested. Pls update your fork:
```shell
git remote add upstream https://github.com/kengz/aiva.git
git fetch upstream
git merge upstream/master
```
or your repo clone:
```
git pull
```

**Comment by manuchis** [comment]
Awesome, tnks

**Comment by kengz** [comment]
Fixed in [commit v4.2.1](https://github.com/kengz/aiva/commit/0758ff274a16e27d835a0707a4884e38c9285280)
